benchmarks/benchmark_noise.py

```diff
@@ -0,0 +1,26 @@
+# See "Writing benchmarks" in the asv docs for more information.
```

Comment:
possibly rename this file to `benchmark_util.py` since most other benchmark files are currently named based on the module they correspond to.